src/styles/product_card.scss

```
@@ -0,0 +1,108 @@
+.maincon {
```

What does `maincon` mean)?
```suggestion
.maincon {
```